src/test/transaction_tests.cpp

```diff
@@ -797,6 +797,17 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     reason.clear();
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "scriptsig-size");
+
+    // Check bare multisig (standard if policy flag fIsBareMultisigStd is set)
+    fIsBareMultisigStd = true;
+    t.vout[0].scriptPubKey = GetScriptForMultisig(2, std::vector<CPubKey>(3, key.GetPubKey())); // 2-of-3
```

Comment:
For this test I think you should do the simplest base case to make it trivially certain you're not hitting some size-based restriction(as I recall, naked multisigs increase virtual size of transactions during relay....)

so just make a 1-of-1 multisig

Reply:
Done, thanks for reviewing.